mne/io/eeglab/eeglab.py

```diff
@@ -119,6 +117,30 @@ def read_raw_eeglab(input_fname, montage=None, preload=False, eog=(),
         Path or instance of montage containing electrode positions.
         If None, sensor locations are (0,0,0). See the documentation of
         :func:`mne.channels.read_montage` for more information.
+    eog : list | tuple | 'auto'
+        Names or indices of channels that should be designated EOG channels.
+        If 'auto', the channel names containing ``EOG`` or ``EYE`` are used.
+        Defaults to empty tuple.
+    misc : list or tuple
```

Comment:
@dengemann you mean 

```RawEEGLAB(input_fname=input_fname, montage=montage, preload=preload,
                          eog=eog, misc=misc, event_id=event_id,
                          event_id_func=event_id_func, verbose=verbose)
```

or the same for `EpochsEEGLAB`
?

But then?.. It's not used, right, including in the `super()`?